src/components/layout/AppHeader.vue

```diff
@@ -3,6 +3,15 @@
 		<a class="navigation-left" :href="`/?${ campaignParams }`" aria-hidden="true">
 			<Logo/>
 		</a>
+		<div class="navigation-right">
```

Comment:
I totally agree with putting this before the language navigation, it makes sense semantically and I can see it helping the accessibility too.
However, I've tested it with VoiceOver in Firefox and Safari and did not see any change to the behavior before - at least regarding the order. To me, it looks like the positioning via CSS takes precedence over the markup position. But maybe I'm using VoiceOver wrong. Could you demonstrate the effect of this change in the next daily?

Reply:
Oh it was working for me with tabbing in FF, I'll do a VoiceOver test